tiatoolbox/utils/visualization.py

```diff
@@ -138,3 +161,45 @@ def overlay_patch_prediction(
     cbar.ax.tick_params(labelsize=12)
 
     return ax
+
+
+def overlay_prediction_contours(
```

Comment:
I actually don’t like the random colours. I think if no type is supplied then we should default to yellow overlay